examples/solidity/basic/default.yaml

```diff
@@ -47,8 +47,6 @@ initialize: null
 multi-abi: false
 #checkAsserts checks assertions
 checkAsserts: false
-#dashboard determines if output is just text or an AFL-like display
-dashboard: true
```

Comment:
This means we cannot disable the dashboard anymore?

Reply:
Dashboard is shown by default, it is disabled automatically when `format:` option is specified. The `dashboard` option turned out to be redundant.